.github/workflows/integration_test.yaml

```diff
@@ -3,20 +3,21 @@ name: Integration tests
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
-
 on:
   pull_request:
   workflow_call:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: arjun11-malik/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      channel: 1.25-strict/stable
+      provider: k8s
+      use-canonical-k8s: true
+      channel: edge
+      charmcraft-channel: latest/edge
```

Comment:
A less risky approach would be to use `latest/candidate` instead, that way we would be using a more stable version. In the past we've hit walls when charmcraft edge breaks.
Another option would be to use a stable version, currently 3.x/stable. Please let me know if you had issues with this.